tst/EnergyPlus/unit/Coils/CoilCoolingDX.unit.cc

```diff
@@ -850,30 +850,30 @@ TEST_F(EnergyPlusFixture, DISABLED_CoilDXCoolingVsMultiSpeed_ContFanCycCoil)
     constantcurve1.curveType = CurveManager::CurveType::BiQuadratic;
     constantcurve1.ObjectType = "Curve:Biquadratic";
     constantcurve1.InterpolationType = CurveManager::InterpType::EvaluateCurveToLimits;
-    constantcurve1.Coeff1 = 1.0;
-    constantcurve1.Coeff2 = 0.0;
-    constantcurve1.Coeff3 = 0.0;
-    constantcurve1.Coeff4 = 0.0;
-    constantcurve1.Coeff5 = 0.0;
-    constantcurve1.Coeff6 = 0.0;
-    constantcurve1.Var1Min = 10.0;
-    constantcurve1.Var1Max = 25.0;
-    constantcurve1.Var2Min = 0.0;
-    constantcurve1.Var2Max = 100.0;
-    constantcurve1.CurveMin = 1.0;
-    constantcurve1.CurveMax = 1.0;
+    constantcurve1.coeff[0] = 1.0;
+    constantcurve1.coeff[1] = 0.0;
+    constantcurve1.coeff[2] = 0.0;
+    constantcurve1.coeff[3] = 0.0;
+    constantcurve1.coeff[4] = 0.0;
+    constantcurve1.coeff[5] = 0.0;
+    constantcurve1.inputLimits[0].min = 10.0;
```

Comment:
I think you can just do `constantCurve1.inputLimits[0] = {10.0, 25.0};`.

Reply:
But then what if later I change the order of arguments in the limits class to be alphabetical (max, min)?  Wouldn't that silently (and confusingly) break things?  The initializer list works really well for ordered lists, but I'm not as sure in this case when Limits is just a full class.  Feel free to convince me.